_usage/mvu.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: default
+title: Performing a major version upgrade
+nav_order: 9
+---
+
+## Performing a major version upgrade
+
+Babelfish supports a major version upgrade (MVU) from implementations deployed with a Babelfish PostgreSQL server version 13.6 or later. Currently, the Babelfish PostgreSQL server only supports an upgrade path using the `pg_upgrade` module. During the upgrade, you will need to:
+
+- Perform a major version upgrade of the PostgreSQL server.
+- Update the Babelfish extensions to the target version.
+- Confirm that the Babelfish-specific GUCs are the same in the source and target versions.
+
+### Prerequisites
+
+Before beginning an upgrade, we strongly recommend that you create a copy of the old PostgreSQL `data` directory in case you have trouble and need to revert to it.
+
+Some Babelfish specific parameter settings are considered critical and their values must be the same before and after an upgrade. Changing the following parameter values during the upgrade might cause inconsistent database behavior or even data corruption. 
+
+- `babelfishpg_tds.tds_default_numeric_precision`
+- `babelfishpg_tds.tds_default_numeric_scale`
+- `babelfishpg_tsql.database_name`
+- `babelfishpg_tsql.default_locale`
+- `babelfishpg_tsql.migration_mode`
+- `babelfishpg_tsql.server_collation_name`
+
+You should note the value of any parameter that you've modified so you can ensure that the parameter is reset to the original value after upgrading. To find the values, connect with psql, and use the `SHOW` command:
+
+```sql
+${BABELFISH_HOME_OLD}/bin/psql -d babelfish_db -U postgres
+
+=> SHOW parameter_name;
+```
+
+Babelfish supports a major version upgrade on Babelfish PostgreSQL servers version 13.6 or later. You can use the following releases as an upgrade source (the server from which you'll upgrade):
+
+- Babelfish 1.2.2, built with PostgreSQL 13.6
+- Babelfish 1.3.1, built with PostgreSQL 13.7
+- Babelfish 1.4.0, built with PostgreSQL 13.8
+- Babelfish 1.5.0, built with PostgreSQL 13.9
+
+You can use the following Babelfish release as an upgrade target (the version to which you'll upgrade):
+
+- Babelfish 2.3.0, built with PostgreSQL 14.6
+
+To identify the Babelfish version you are currently using, you can query the Babelfish server with psql:
+
+```sql
+${BABELFISH_HOME_OLD}/bin/psql -d babelfish_db -U postgres
+
+=> select sys.SERVERPROPERTY('BabelfishVersion') AS Babelfish_Version;
+  babelfish_version
+--------------------
+       1.2.1
+```
+
+If you are not running a version listed in the source server list, you should first perform a minor version upgrade to a supported version. To perform a minor version upgrade, follow the instructions listed in the `INSTALLING.md` file included in the download of the release to which you are upgrading. After upgrading the server, run the following commands to update the extensions:
```

Comment:
Updated to read:
If you are not running a version listed in the source server list, you should first perform a minor version upgrade to a supported version. To perform a minor version upgrade, follow the instructions listed in the `INSTALLING.md` file included in the download of the release to which you are upgrading; this will build and replace the earlier version with new binaries and libraries. After upgrading the server, use the following commands to update the extensions. 

You should run these commands in a single transaction/scope to avoid inconsistency in the event of failures: